worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -120,7 +120,12 @@ private MemoryManager(CelebornConf conf) {
             .<Long>invoke();
 
     Preconditions.checkArgument(maxDirectorMemory > 0);
-    Preconditions.checkArgument(pauseReplicateRatio > pausePushDataRatio);
+    Preconditions.checkArgument(
+        pauseReplicateRatio > pausePushDataRatio,
+        String.format(
+            "Invalid config, {} should be greater than {}",
```

Comment:
let's also mention the value in the error message
```suggestion
            "Invalid config, {}({}) should be greater than {}({})",
```